File: demo-wiremock/src/test/java/io/fries/wiremock/demo/wiremock/cucumber/wiremock/MockServers.java
package io.fries.wiremock.demo.wiremock.cucumber.wiremock;

import java.util.List;

public class MockServers implements MockServer {

    private final List<MockServer> servers;

    public MockServers(final List<MockServer> servers) {
        this.servers = servers;
    }

    @Override
    public void start() {
        servers.forEach(MockServer::start);
    }

    @Override
    public void stop() {
        servers.forEach(MockServer::stop);
    }
}

File: examples/demo-cucumber/src/test/kotlin/io/fries/proxito/demo/cucumber/glue/wiremock/MockServers.kt
package io.fries.proxito.demo.cucumber.glue.wiremock

class MockServers(
    private val servers: List<MockServer> = listOf()
) : MockServer {

    override fun start() {
        servers.forEach(MockServer::start)
    }

    override fun stop() {
        servers.forEach(MockServer::stop)
    }
}